app/models/user.rb

```diff
@@ -1,13 +1,21 @@
 # frozen_string_literal: true
 # User model class for all user types.
 class User < ApplicationRecord
+  # Determine whether or not CAS authentication is being used
+  #
```

Comment:
we should probably note that this must be at the top of the class in order for the devise conditional to work